### PR TITLE
add some new methods to UImage+Edit

### PR DESCRIPTION
**Pull request by ChachiQ**
add some new methods to UImage+Edit

### CODE REVIEW

File: EFCore/EFCategory/UIImage/UIImage+Edit.h [hunk 1]
```diff
@@ -16,4 +17,14 @@
 - (UIColor *)averageColor;
 
 - (UIColor *)averageColorForRect:(CGRect)rect;
+
+- (UIImage *)subImageForRect:(CGRect)rect;
```
Comment (yzlhccdec) [inline EFCore/EFCategory/UIImage/UIImage+Edit.h]: - (UIImage *)cropToBounds:(CGRect)bounds

File: EFCore/EFCategory/UIImage/UIImage+Edit.h [hunk 2]
```diff
@@ -16,4 +17,14 @@
...
+
+- (UIImage *)scaleToSize:(CGSize)size;
```
Comment (yzlhccdec) [inline EFCore/EFCategory/UIImage/UIImage+Edit.h]: resizedImage

Comment (ChachiQ) [inline EFCore/EFCategory/UIImage/UIImage+Edit.h]: resized感觉不太好诶，做了scale操作

File: EFCore/EFCategory/UIImage/UIImage+Edit.h [hunk 3]
```diff
@@ -16,4 +17,14 @@
...
+
+- (UIImage *)scaleCropInCenterToSize:(CGSize)size;
+
+- (UIImage *)scaleToWidth:(CGFloat)width;
```
Comment (yzlhccdec) [inline EFCore/EFCategory/UIImage/UIImage+Edit.h]: scaleToFillWidth

File: EFCore/EFCategory/UIImage/UIImage+Edit.h [hunk 4]
```diff
@@ -16,4 +17,14 @@
 - (UIColor *)averageColor;
 
 - (UIColor *)averageColorForRect:(CGRect)rect;
+
+- (UIImage *)subImageForRect:(CGRect)rect;
+
+- (UIImage *)scaleToSize:(CGSize)size;
+
+- (UIImage *)scaleCropInCenterToSize:(CGSize)size;
```
Comment (yzlhccdec) [inline EFCore/EFCategory/UIImage/UIImage+Edit.h]: (UIImage *)cropToFillSize:(CGSize)size

**Pull request merged**